plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java

```diff
@@ -200,6 +227,10 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @description
         // Returns each block location within the area.
         // Optionally, specify a material matcher to only return locations with that block type.
+        // @example
+        // # Narrates the locations of blocks that match "*planks" within the area.
+        // - foreach <cuboid[my_cuboid].blocks[*planks]> as:plank:
+        //      - narrate <[plank]>

```

Comment:
This is narrating a raw location. Should use a more appropriate example behavior, perhaps a `debugblock`